pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -0,0 +1,52 @@
+ifdef::context[:parent-context: {context}]
+
+include::../attributes.adoc[]
+
+[id="help"]
+= Help
+
+:context: assembly-pantheon-help
+
+Welcome to Pantheon help.
+
+This document describes the main concepts, requirements and, actions in Pantheon.
```

Comment:
requirements and, => requirements, and

I saw that Ashwin used an attribute for a placeholder for "Pantheon". Should we all be doing that?

Reply:
Changed as requested.

In regards to the use of attributes, it probably makes sense to do so, and I've updated all references to 'Pantheon' to use the attribute version instead. I've also added this to our review issues.